wsmaster/che-core-api-workspace/src/test/resources/devfile/schema_test/command/devfile_command_with_preview_url.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+---
+apiVersion: 1.0.0
+metadata:
+  name: petclinic-dev-environment
+components:
+  - alias: mysql
+    type: openshift
+    reference: petclinic.yaml
+    selector:
+      app.kubernetes.io/name: mysql
+      app.kubernetes.io/component: database
+      app.kubernetes.io/part-of: petclinic
+commands:
+  - name: build
+    previewUrl:
+      port: 65535
```

Comment:
It seems to be invalid devfile. I mean that openshfit component is not able to contains endpoints, so it won't produce server and such previewURL can not be evaluated. 
So:
1. Let's avoid non-working examples even in test sources, it would be better to use dockerimage component.
2. Maybe we should implement some validation... The difficulty that I see here - referenced plugins/editor configuration should be downloaded for validations since they can contain endpoints.

Reply:
current implementation will create service+ingress in case it won't find ones with port